chromadb/utils/rendezvous_hash.py

```diff
@@ -0,0 +1,29 @@
+# An implementation of https://en.wikipedia.org/wiki/Rendezvous_hashing
+from typing import Callable, cast
```

Comment:
Note that the go code and the python code never have to agree on a rendezvous hash so its ok that we don't cross-test for now.